packages/xrpl/test/integration/submitAndWaitBatch.test.ts

```diff
@@ -0,0 +1,481 @@
+      Amount: '1000',
+    }
+    const txList = [
+      {
+        transaction: invalidAccountDeleteTx,
+        opts: { wallet: this.wallet },
+      },
+    ]
+
+    const responsePromise = this.client.submitAndWaitBatch(txList)
+
+    const ledgerPromise = setTimeout(ledgerAccept, 1000, this.client)
+    return Promise.all([responsePromise, ledgerPromise]).then(
+      ([result, _ledger]) => {
+        assert.equal(result.success.length, 0)
+        assert.equal(result.error.length, 1)
+        assert.equal(result.unsubmitted.length, 0)
+        assert.equal(result.error[0].data.error, 'invalidTransaction')
+        assert.equal(result.error[0].data.status, 'error')
+      },
+    )
+  })
+
+  it('submitAndWaitBatch a single account submits multiple payment transactions', async function () {
+    const receiverWallet = await generateFundedWallet(this.client)
+    const receiverWallet2 = await generateFundedWallet(this.client)
+
+    const paymentTx: Payment = {
+      TransactionType: 'Payment',
+      Account: this.wallet.classicAddress,
+      Destination: receiverWallet.classicAddress,
+      Amount: '1000',
+    }
+    const paymentTx2: Payment = {
+      TransactionType: 'Payment',
+      Account: this.wallet.classicAddress,
+      Destination: receiverWallet2.classicAddress,
+      Amount: '1000',
+    }
+    const txList = [
+      {
+        transaction: paymentTx,
+        opts: { wallet: this.wallet },
+      },
+      {
+        transaction: paymentTx2,
+        opts: { wallet: this.wallet },
+      },
+    ]
+
+    const responsePromise = this.client.submitAndWaitBatch(txList)
+
+    const ledgerPromise = setTimeout(ledgerAccept, 1000, this.client)
+    const ledgerPromise2 = setTimeout(ledgerAccept, 3000, this.client)
+    return Promise.all([responsePromise, ledgerPromise, ledgerPromise2]).then(
+      ([result, _ledger, _ledger2]) => {
+        assert.equal(result.success.length, 2)
+        assert.equal(result.error.length, 0)
+        assert.equal(result.unsubmitted.length, 0)
+        for (const response of result.success) {
+          assert.equal(response.type, 'response')
+          assert.equal(response.result.validated, true)
+        }
+      },
+    )
+  })
+
+  it('submitAndWaitBatch a single account submits multiple payment transactions with one failed transaction', async function () {
+    const receiverWallet = await generateFundedWallet(this.client)
+
+    const paymentTx: Payment = {
+      TransactionType: 'Payment',
+      Account: this.wallet.classicAddress,
+      Destination: receiverWallet.classicAddress,
+      Amount: '1000',
+    }
+    const invalidAccountDeleteTx = {
+      TransactionType: 'AccountDelete',
+      Account: this.wallet.classicAddress,
+      Destination: this.wallet.classicAddress,
+      Amount: '1000',
+    }
+    const txList = [
+      {
+        transaction: paymentTx,
+        opts: { wallet: this.wallet },
+      },
+      {
+        transaction: invalidAccountDeleteTx,
```

Comment:
What happens if you submit multiple transactions with a failed one in the middle of the order?

Reply:
For a single account, the failed one and subsequent transactions (they will also fail) will be returned in the `result.error` list.

Reply:
The subsequent ones won't fail if you're using tickets, which should be recommended in the docs for this method.